### ISSUE TITLE
Hard to fix stylings

### ISSUE THREAD

**Issue by ZhenWang-Jen** [ZhenWang-Jen/Ras-Signaling#1]
@pwwang Hi Panwen, I was working on finalizing my web page for the ras signaling pathway this week. Unfortunately, I have been very struggling with the style. I.e. those links cover parts of nodes which make the text not readable somehow. I tried adding paddings/margins on nodes but it didn't work well. Do you know how to make the web page presentable? I appreciate it!

**Comment by pwwang** [comment]
Could you make a snapshot and point it out?

**Comment by ZhenWang-Jen** [comment]
Thanks Panwen. In the screenshot, one thing is I've adjusted the arrows and links for multiple times but it still not fit for every node. For example, if things look good for nodes in blue rectangle, some others like those in red circle will look bad. 

I also tried to add some paddings to nodes so the arrows won't cover the node text. But I still feel hard to fix them.

All code for this project is uploaded here. I'm currently interning at an IT firm until mid June. I'll send my slides of my understanding of Ras and my web demo to you and Dr. Wang shortly. 
<img width="1436" alt="issue" src="https://user-images.githubusercontent.com/28586492/37978567-349c84c6-319b-11e8-8aa0-f77b9387b6dd.png">

I appreciate your help.
